src/kde/kde.i.hh

```diff
@@ -0,0 +1,76 @@
+//--------------------------------------------*-C++-*---------------------------------------------//
+/*!
+ * \file   kde/kde.i.hh
+ * \author Mathew Cleveland
+ * \date   Nov. 10th 2020
+ * \brief  Member definitions of class kde
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//------------------------------------------------------------------------------------------------//
+
+// clang-format off
+
+#ifndef kde_kde_i_hh
+#define kde_kde_i_hh
+
+#include "ds++/dbc.hh"
+
+namespace kde {
+
+  //! DEFAULT constructor to return error if instantiation is not found
```

Comment:
* I would avoid the term "constructor" in this context.
* Do you know all of the valid `<coord, dim>` combinations?  If so, it is better to prevent this function from ever being instantiated.

Reply:
fixed.